kubernetes-deploy-main.tpl

```diff
@@ -48,10 +48,10 @@ kind: Ingress
 metadata:
   name: prototype-ingress-${BRANCH}
   annotations:
-    kubernetes.io/ingress.class: nginx
     external-dns.alpha.kubernetes.io/set-identifier: prototype-ingress-${BRANCH}-${KUBE_NAMESPACE}-green
     external-dns.alpha.kubernetes.io/aws-weight: "100"
 spec:
+  ingressClassName: default OR modsec
   tls:
```

Comment:
Is `ingressClassName` a genuine part of the Deployment spec schema? Didn't see this on any Deployments we have currently, so was wondering if this was only a GenericService thing?
e.g. 
`kubectl --context CP2 -n activities-api-dev get deployment hmpps-activities-management-api -o yaml | grep ingressClass`  doesn't match anything.

Reply:
Sorry - found it on the Ingress, not the deployment. Ignore the above.